libs/config/src/v4.rs

```diff
@@ -898,4 +907,68 @@ mod tests {
             .unwrap();
         assert_eq!(res.ip, Ipv4Addr::new(192, 168, 0, 120));
     }
+
+    /// test if v6_config can generate a server_id; and if it can dump it to a file
+    #[test]
```

Comment:
I put this code here because the v6 configuration was nested in the v4 structure in the original code. Anyway, it makes more sense to put the v6 function test in the v6 file. So I will make the changes as you suggested.